.circleci/config.yml

```diff
@@ -123,11 +126,20 @@ jobs:
           name: Run Reg Test
           command: |
             cd ~/ExamplesRegtest
-            command="python3.11 RunNightlyTest.py --path ../repo --conf_ignore conf_ignore.txt --json_flags '{\"run_source\": \"LIGHT_TEST\", \"disable_auto_cache_key_gen\": true, \"group_id\": \"$CIRCLE_WORKFLOW_ID\"}' --send_message --disable_cleanup"
+            python3.11 CopyExpected.py
+            command="Regression --path << parameters.examples_repo_path >> --conf_ignore conf_ignore.txt --json_flags '{\"run_source\": \"LIGHT_TEST\", \"disable_auto_cache_key_gen\": true, \"group_id\": \"$CIRCLE_WORKFLOW_ID\"}' --send_message --checks Verification"
             if [ "<< parameters.certora_cli_package >>" == "certora-cli-alpha-master" ]; then
               command="$command --prover_version master"
             fi
             eval $command
+ 
+      - run:
+          name: Install ReportAnalysis package and execute ReportAnalyzer 
+          command: |
+            pip3.11 install git+ssh://git@github.com-reportanalysis/Certora/ReportAnalysis.git
+            ReportAnalyzer --ci_client "Examples Nightly"
```

Comment:
Why not use << parameters.certora_ci_name >>?

Reply:
Analyzer should only work on the nightly workflow.

Reply:
I agree, but in case we change the name for some weird reason, it would be nice to have it generic.